src/IssueViz.Security/HotspotsList/HotspotsControl.xaml

```diff
@@ -277,55 +278,55 @@
                         </DataTemplate>
                     </DataGridTemplateColumn.CellTemplate>
                 </DataGridTemplateColumn>
-                <DataGridTemplateColumn CanUserSort="True" SortMemberPath="Hotspot.Issue.RuleKey" Header="Code" Width="130">
+                <DataGridTemplateColumn CanUserSort="True" SortMemberPath="Hotspot.Issue.RuleKey" Header="Code" Width="130" MinWidth="50">
                     <DataGridTemplateColumn.CellTemplate>
                         <DataTemplate>
                             <TextBlock Text="{Binding Hotspot.Issue.RuleKey}" 
                                        Style="{StaticResource CellTextBlockStyle}" />
                         </DataTemplate>
                     </DataGridTemplateColumn.CellTemplate>
                 </DataGridTemplateColumn>
-                <DataGridTemplateColumn CanUserSort="True" SortMemberPath="CategoryDisplayName" Header="Category" Width="Auto">
+                <DataGridTemplateColumn CanUserSort="True" SortMemberPath="CategoryDisplayName" Header="Category" Width="Auto"  MinWidth="50">
```

Comment:
MinWidth is a bit small

![image](https://user-images.githubusercontent.com/60586879/101141688-ef7b1980-3614-11eb-8027-670fc99e0fcc.png)

Reply:
I was setting the minimum widths so the columns are "more or less usable" if set to a minimum width. I wasn't trying to set them so that all of the text was always visible, or the sort arrows.
However, I'll increase the minimums slightly.